### PR TITLE
Milestone 2 components and redux store:PR

### PR DESCRIPTION
**Pull request by kenderb**
## Milestone 2: components and redux store
In this branch we:
- Prepared the directory structure.
- Implemented the App component.
- Implemented the BooksList component.
- Implemented the Book component.
- Implemented  the BooksForm component.
- Defined the Redux actions and reducers.
- Initialized  the Redux store.

### CODE REVIEW

File: src/components/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <Books
+          id={book.id}
+          key={book.id}
+          title={book.title}
+          category={book.category}
+        />
+      ))}
+    </table>
+  );
+};
```
Comment (bellom) [inline src/components/BooksList.js]: Welldone 👍 

Let's update the code.

You need to do the following;
1. Improve the file structure: `BookList` should be placed in `src/containers` directory.
2. `BookList` should connect to Redux store and fetches books from the state using [mapStateToProps](https://react-redux.js.org/using-react-redux/connect-mapstate) [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToRedux.png)

File: src/components/BooksForm.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const categoryList = [
+  'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi',
+];
+
+const BooksForm = () => (
```
Comment (bellom) [inline src/components/BooksForm.js]: Improve the file structure: `BooksForm` should be placed in `src/containers` directory.

image
[Microverse Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by Torres-ssf — APPROVED** [review]
### Milestone Approved :+1: 
Hi team :hand: 
Awesome job on this milestone.
Good luck with the next one.

Happy coding :computer: 
**Sérgio Torres**

**Comment by ghost** [comment]
> ### Milestone Approved 👍
> Hi team ✋
> Awesome job on this milestone.
> Good luck with the next one.
> 
> Happy coding 💻
> **Sérgio Torres**

thanks

**Pull request merged**